BestFloww/BestFlow
# set up simple BE interaction

## Pull Request Summary
Added a quick FE/BE interaction button for the demo. I also added axios for API calls.

### Change types


### Testing

I have not added testing since we will delete this soon.

### How to manually test this PR (QA instructions)
npm install and then run both FE/BE. Press the third button and ensure hello world is printed to console.

## Questions
